sentry_sdk/integrations/django/__init__.py

```diff
@@ -358,9 +363,33 @@ def record_sql(sql, params, cursor=None):
         except Exception:
             pass
 
+    span = None
+
     if real_sql:
         with capture_internal_exceptions():
             hub.add_breadcrumb(message=real_sql, category="query")
+            span = hub.start_span(op="sql.query", description=real_sql)
+
+    if span is None:
+        yield
+    else:
+        try:
+            yield
+        finally:
+            span.set_tag("status", sys.exc_info()[1] is None)
```

Comment:
Please use `with`.

Reply:
Btw `OT` calls this `error`
https://github.com/opentracing/specification/blob/master/semantic_conventions.md

Reply:
Fixed both (one fix in in hub.py)